### PR TITLE
Adds a line about nosources

### PR DESCRIPTION
**Pull request by pksjce**
With reference to #133

### CODE REVIEW

**Comment by SpaceK33z** [comment]
Do you know what the difference is with this and passing `false`?

File: content/configuration/devtool.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ W> There are some issues with Source Maps in Chrome. [We need your help!](https:
 
 `cheap-module-source-map` - A SourceMap without column-mappings that simplifies loaded Source Maps to a single mapping per line.
 
+`nosources` - This option prevents the generation of a SourceMap file for your project.
```
Comment (sokra) [inline content/configuration/devtool.md]: the description is wrong. it generates a SourceMap but don't include the sources in the SourceMap.

Comment (pksjce) [inline content/configuration/devtool.md]: I tried this out in my project
 - `devtool: 'nosources'` - No source map files were created.
 - `devtool: false` - No sourcer
 - `devtool : <Any other option>` - Source map file is created

Should it behave differently?

**Comment by pksjce** [comment]
@sokra - How is the sourcemap used if it doesnt have `sourcesContent`? Is there a way to provide `sourceRoot` in the webpack config?

**Comment by sokra** [comment]
A SourceMap without `sourcesContent` can be used to map stack traces on client without exposing too much source code.

**Comment by bebraw** [comment]
@pksjce Good to go?

**Comment by pksjce** [comment]
@bebraw @sokra I think so, Checked all the cases in [devtool-example](https://github.com/kalcifer/webpack-examples/tree/master/devtool)

**Pull request merged**